# Synchronise timestamps for cameras
#	stamp: timestamp
time timeStamp

# This can be "full" or "preview"
string data
